### PR TITLE
cli(engine): optimize tiflow cli

### LINKED ISSUE #6600: tiflow cli command should be optimized
1. cerate job should have `--job-id` option
```bash
/ # /tiflow  cli  job  create  --master-addrs=127.0.0.1:10240 "--job-type=FakeJob" --job-config=/tmp/conf
ig.toml --job-id=test-basic-fake-job
Error: unknown flag: --job-id
Usage:
  tiflow cli job create [flags]

Flags:
  -h, --help                help for create
      --job-config string   path of config file for the job
      --job-type string     job type

Global Flags:
  -i, --interact               Run cdc cli with readline
      --master-addrs strings   server master addresses
      --project-id string      the project id
      --rpc-timeout duration   default rpc timeout (default 30s)
      --tenant-id string       the tenant id
```

2. `--job-type` should follow with detailed job types prompt
```bash
Flags:
  -h, --help                help for create
      --job-config string   path of config file for the job
      --job-type string     job type
```

3. `--master-addrs` should set default value, such as: `127.0.0.1:10240`
```bash
/ # /tiflow  cli  job  create "--job-type=FakeJob" --job-config=/tmp/config.toml 
Error: [DFLOW:ErrInvalidCliParameter]invalid cli parameters: [DFLOW:ErrInvalidCliParameter]master-addrs can't be nil
Usage:
  tiflow cli job create [flags]

Flags:
  -h, --help                help for create
      --job-config string   path of config file for the job
      --job-type string     job type

Global Flags:
  -i, --interact               Run cdc cli with readline
      --master-addrs strings   server master addresses
      --project-id string      the project id
      --rpc-timeout duration   default rpc timeout (default 30s)
      --tenant-id string       the tenant id

[DFLOW:ErrInvalidCliParameter]invalid cli parameters: [DFLOW:ErrInvalidCliParameter]master-addrs can't be nil
```

### PR DESCRIPTION
**Pull request by CharlesCheung96**
### What problem does this PR solve?
Issue Number: close #6600

### What is changed and how it works?
add more prompt info to `tiflow cli`

#### Tests 

 - Unit test
 - Manual test (add detailed scripts or steps below)

### Release note 

```release-note
 `None`.
```

### CODE REVIEW

File: engine/pkg/cmd/cli/cli_job.go [hunk 1]
```diff
@@ -61,12 +60,13 @@ func (o *jobGeneralOptions) addFlags(cmd *cobra.Command) {
 // validate checks that the provided job options are valid.
 func (o *jobGeneralOptions) validate(ctx context.Context, cmd *cobra.Command) error {
 	if len(o.masterAddrs) == 0 {
-		return errors.ErrInvalidCliParameter.GenWithStack("master-addrs can't be nil")
+		o.masterAddrs = []string{"127.0.0.1:10240"}
+		log.Warn("master-addrs is nil, use default addr: 127.0.0.1:10240")
```
Comment (lonng) [inline engine/pkg/cmd/cli/cli_job.go:64]: `nil` is a programmatic term. It's better to use `The master address does not assign, ...`

Comment (CharlesCheung96) [reply engine/pkg/cmd/cli/cli_job.go]: Good suggestion. Already fix it, PTAL.

File: engine/pkg/cmd/cli/cli_job.go [hunk 2]
```diff
@@ -61,12 +61,13 @@ func (o *jobGeneralOptions) addFlags(cmd *cobra.Command) {
 // validate checks that the provided job options are valid.
 func (o *jobGeneralOptions) validate(ctx context.Context, cmd *cobra.Command) error {
 	if len(o.masterAddrs) == 0 {
-		return errors.ErrInvalidCliParameter.GenWithStack("master-addrs can't be nil")
+		o.masterAddrs = []string{"127.0.0.1:10240"}
+		log.Warn("the master-addrs are not assigned, use default addr: 127.0.0.1:10240")
```
Comment (amyangfei) [inline engine/pkg/cmd/cli/cli_job.go:65]: IMO we can add a global variable with default value, and use it as the default value in command flags.

Comment (CharlesCheung96) [reply engine/pkg/cmd/cli/cli_job.go]: PTAL

**Review by amyangfei — APPROVED** [review]

File: engine/pkg/cmd/cli/cli_job.go [hunk 3]
```diff
@@ -22,11 +22,14 @@ import (
...
 	"github.com/spf13/cobra"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
 
 	"github.com/pingcap/tiflow/engine/pkg/tenant"
-	"github.com/pingcap/tiflow/pkg/errors"
 )
 
+// DefaultMasterAddr is the default master address.
+const DefaultMasterAddr = "127.0.0.1:10240"
```
Comment (amyangfei) [inline engine/pkg/cmd/cli/cli_job.go:31]: Don't need to expose this variable.

**Review by lonng — APPROVED** [review]

**Pull request merged**